domain/src/main/kotlin/no/nav/su/se/bakover/domain/oppgave/OppgaveConfig.kt

```diff
@@ -54,6 +54,18 @@ sealed class OppgaveConfig {
         override val oppgavetype = Oppgavetype.BEHANDLE_SAK
     }
 
+    data class Forhåndsvarsling(
+        val saksnummer: Saksnummer,
+        override val aktørId: AktørId,
+        override val tilordnetRessurs: NavIdentBruker? = null
+    ) : OppgaveConfig() {
+        override val saksreferanse = saksnummer.toString()
+        override val journalpostId: JournalpostId? = null
+        override val behandlingstema = Behandlingstema.SU_UFØRE_FLYKNING
+        override val behandlingstype = Behandlingstype.REVURDERING
+        override val oppgavetype = Oppgavetype.FREMLEGGING
```

Comment:
Finner vel fort nok ut om det kreves noe arbeid for å få lagt inn kobling mellom tema/type/oppgavetype i oppgavemapper.